src/video/wayland/SDL_waylandwindow.c

```diff
@@ -1888,6 +1888,13 @@ int Wayland_CreateWindow(_THIS, SDL_Window *window)
     }
 #endif /* SDL_VIDEO_DRIVER_WAYLAND_QT_TOUCH */
 
+    if (window->flags & SDL_WINDOW_TRANSPARENT) {
+        if (_this->gl_config.alpha_size == 0) {
+            _this->gl_config.alpha_size = 1;
+        }
+        SDL_SetHint(SDL_HINT_VIDEO_EGL_ALLOW_TRANSPARENCY, "1");
```

Comment:
Should this hint be removed and its functionality can used by the new window flag? Otherwise it feels sort of redundant.

Reply:
yes, probably, I wasn't sure. so I didn't touch it. this can be done on a separate part, so that this can be also re-added.

Reply:
(Also: 1 bit of alpha in this case?)

Reply:
I changed to 8 which is probably more standard. anyway, if alpha_size is explicitly set, one could request a 1555 or 4444

Reply:
I've just remove the hint SDL_HINT_VIDEO_EGL_ALLOW_TRANSPARENCY